app/services/hyrax/custom_queries/navigators/find_files.rb

```diff
@@ -0,0 +1,29 @@
+module Hyrax
+  module CustomQueries
+    module Navigators
+      class FindFiles
+        # Use:
+        #   Hyrax.query_service.custom_queries.find_files(resource: file_set_resource)
```

Comment:
can we make this a YARD annotation (`@example`) on the class?

Reply:
All the custom queries use this for their comments.  I agree that YARD annotation would be better.  

I'd like to accept this PR with the Use: comment and create an issue to change this in all the custom queries

Reply:
there are existing PRs to change the other queries. see, e.g. #4078 

i'd strongly prefer not to need to go back and do this one separately.

Reply:
DONE - changed to @example in all custom queries effected by this PR